runtime-sdk/src/dispatcher.rs

```diff
@@ -28,13 +28,20 @@ use crate::{
     modules,
     modules::core::API as _,
     runtime::Runtime,
-    storage, types,
-    types::transaction::TransactionWeight,
+    storage,
+    storage::Prefix,
+    types,
+    types::transaction::{Transaction, TransactionWeight},
 };
 
 /// Unique module name.
 const MODULE_NAME: &str = "dispatcher";
 
+const PREFETCH_LIMIT: u16 = std::u16::MAX;
+
+/// Local config key for disabling prefetch.
+const CFG_PREFETCH_DISABLE: &str = "prefetch_disable";
```

Comment:
Does this make sense, or would it made more sense for this to be configurable by the runtime?

Reply:
The only problem can be if the value of this could affect security and/or determinism in any way. On the other hand making it only configurable by the runtime would make it impossible for node operators to turn this off and would ensure that all runtime instances have it configured the same.

What is the reason one would want to disable prefetching?

Reply:
Yeah actually maybe indeed there's no reason for disabling prefetching. It was mostly useful for me to comapre benchmarks with and without it - but that can also be done by making it configurable by the runtime. It affects the account transfer benchmarks for about 5% (prefetch worsens it).

Reply:
I made `PREFETCH_LIMIT` a Runtime trait constant, and if set to `0` it will disable prefetch. I think this makes most sense (makes the limit configurable by runtimes and also can be used to disable prefetch).